tools/generators/epic-generator/index.spec.ts

```diff
@@ -0,0 +1,275 @@
+import { addProjectConfiguration, ProjectType, serializeJson, stripIndents } from '@nrwl/devkit';
+import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing';
+import { execSync } from 'child_process';
+import epicGenerator from './index';
+import { readFileSync } from 'fs';
+
+jest.mock('child_process');
+// @ts-ignore
+const execSyncMock = <jest.Mock<string>>execSync;
+
+jest.mock('fs');
+// @ts-ignore
+const readFileSyncMock = <jest.Mock<string>>readFileSync;
+
+type Package = {
+  name: string;
+  version: string;
+  projectType: ProjectType;
+  owners: string[];
+};
+function setupTest(packages: Package[]) {
+  const tree = createTreeWithEmptyWorkspace();
+
+  // Initialize NX package structure
+  packages.forEach(pckg => {
```

Comment:
**suggestion:** `pckg` ? Im familiar with `pkg` abbreviation. ideally we should use human readable names

Reply:
`package` is a keyword 🥲 I'll use `pkg` instead.